starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,76 @@
+*{
+    font-family: sans-serif
+}
+body{
+    margin: 0;
+}
+
+.first-section{
+   
+    background-image: url(file:///Users/alejandrogomez/Desktop/Programs/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg);
```

Comment:
Make sure you use relative pathing when you're accessing an image file. In this case, you would use url("../images/apple-pie.jpg") to render the image. Something to keep in mind!